### ISSUE TITLE
test

### ISSUE THREAD

**Issue by bernt-matthias** [Helmholtz-UFZ/galaxy-tools#33]
# Test Results (powered by Planemo)

## Test Summary

<div class="progress">
  <div class="progress-bar progress-bar-success" style="width: 100.0%" aria-valuenow="18" aria-valuemin="0" aria-valuemax="18" data-toggle="tooltip" title="18 Passed">
  </div>
  <div class="progress-bar progress-bar-warning" style="width: 0.0%" aria-valuenow="0" aria-valuemin="0" aria-valuemax="18" data-toggle="tooltip" title="0 Skipped">
  </div>
  <div class="progress-bar progress-bar-danger" style="width: 0.0%" aria-valuenow="0" aria-valuemin="0" aria-valuemax="18" title="0 Failed or Errored">
  </div>
</div>

| Test State | Count |
| ---------- | ----- |
| Total      | 18 |
| Passed     | 18 |
| Error      | 0 |
| Failure    | 0 |
| Skipped    | 0 |

<details ><summary>Passed Tests</summary>

* <summary class="light-green">&#9989; sqlite_to_tabular (Test # 1)</summary>

* <summary class="light-green">&#9989; filter_tabular (Test # 1)</summary>

* <summary class="light-green">&#9989; filter_tabular (Test # 2)</summary>

* <summary class="light-green">&#9989; filter_tabular (Test # 3)</summary>

* <summary class="light-green">&#9989; filter_tabular (Test # 4)</summary>

* <summary class="light-green">&#9989; filter_tabular (Test # 5)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 1)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 2)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 3)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 4)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 5)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 6)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 7)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 8)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 9)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 10)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 11)</summary>

* <summary class="light-green">&#9989; query_tabular (Test # 12)</summary>

</details>
